### PR TITLE
IPFS: Update disclaimers about Private Beta and SSL certificates

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses https://jira.cfops.it/browse/PCX-2750.

### CODE REVIEW

**Comment by thibmeu** [comment]
lgtm.

**Review by thibmeu — APPROVED** [review]

File: products/distributed-web/src/content/ipfs-gateway/connecting-website.md [hunk 1]
```diff
@@ -133,9 +133,11 @@ recommended course.
 
 ## Make It All Secure
 
-> To improve our service, the IPFS gateway has moved to a Private Beta.
-> During this period, we are not offering self-provisioning of SSL cerificates.
-> You [can register](https://forms.gle/3c2xAzawnDcqWzgN7) to get notified once this service opens up.
+<Aside type="warning">
+
+During the Private Beta, self-provisioning of SSL certificates is not available.
```
Comment (marciocloudflare) [inline products/distributed-web/src/content/ipfs-gateway/connecting-website.md:138]: Any reason `Private Beta` is in caps?

File: products/distributed-web/src/content/ipfs-gateway/index.md [hunk 2]
```diff
@@ -13,6 +13,12 @@ Cloudflare's gateway is hosted at https://cloudflare-ipfs.com/. You can find the
 basics on IPFS and [how to serve your website](/ipfs-gateway/connecting-website) through our gateway there. These
 docs are going to get further into the weeds.
 
+<Aside type="warning">
+
+To improve our service, the IPFS Gateway is now available in Private Beta. You can [register](https://forms.gle/3c2xAzawnDcqWzgN7) to get notified when the service opens to a wider audience.
```
Comment (marciocloudflare) [inline products/distributed-web/src/content/ipfs-gateway/index.md:18]: If you change `Private Beta` above, do it here as well :)

**Review by marciocloudflare — APPROVED** [review]

File: products/distributed-web/src/content/ipfs-gateway/connecting-website.md (hunk 1 above)
Comment (pedrosousa) [reply products/distributed-web/src/content/ipfs-gateway/connecting-website.md:138]: We also use "in Beta" for other products... I'll keep it for now, but I'm willing to revisit later in a separate ticket.

**Pull request merged**